app/permissions_migrations.go

```diff
@@ -69,6 +69,7 @@ const (
 	PERMISSION_READ_PUBLIC_CHANNEL_GROUPS        = "read_public_channel_groups"
 	PERMISSION_READ_PRIVATE_CHANNEL_GROUPS       = "read_private_channel_groups"
 	PERMISSION_EDIT_BRAND                        = "edit_brand"
+	PERMISSION_MANAGE_SHARED_CHANNELS            = "manage_shared_channels"
```

Comment:
Why do we always re-add permission names here rather than just using, say, `PERMISSION_MANAGE_SHARED_CHANNELS.Id` in the migration functions?

Reply:
Absolutely no reason other than to maintain consistency with the old style. We could do it of course, and I am 0/5 on it. Let me know what you prefer.

Reply:
I have also added to this pattern recently. I think the next person to touch this (likely someone from Enterprise) can take care of cleaning it up, no need to do it as part of this PR. I wasn't sure if I was not seeing a reason.